protocol/contracts/dao/Comptroller.sol

```diff
@@ -77,8 +77,10 @@ contract Comptroller is Setters {
         uint256 totalRedeemable = totalRedeemable();
         uint256 totalCoupons = totalCoupons();
         if (totalRedeemable < totalCoupons) {
+
             newRedeemable = totalCoupons.sub(totalRedeemable);
-            newRedeemable = newRedeemable > newSupply ? newSupply : newRedeemable;
+            rewardAdjustedSupply = newSupply.mul(getCouponSupplyRatio()).div(100);
```

Comment:
nit: maybe `weightedCouponNewSupply` or similar would be a clearer variable name

Reply:
Hey Joey thanks, not a nit, I am more than happy to change the variable names and that is a good one. Kinda been thinking about that on the side all weekend but was more worried about making sure I had other details right. Thanks for the suggestion!